lib/browser/lighthouse.js

```diff
@@ -42,59 +43,52 @@ function getTotalScore(success, failure) {
   return (successCount * 100) / (successCount + errorCount);
 }
 
-const lhConfig = {
-  passes: [
-    {
-      passName: 'accessibilityPass',
-      gatherers: ['accessibility'],
+const defaultConfig = {
+  extends: 'lighthouse:default',
+  settings: {
+    onlyCategories: [],
+    throttlingMethod: 'simulate',
+  },
+};
+
+const categoryConfig = {
+  performance: {
+    settings: {
+      skipAudits: ['final-screenshot', 'is-on-https', 'screenshot-thumbnails'],
     },
-  ],
-  audits: [
-    'accessibility/manual/accesskeys',
-    'accessibility/aria-allowed-attr',
-    'accessibility/aria-required-attr',
-    'accessibility/aria-required-children',
-    'accessibility/aria-required-parent',
-    'accessibility/aria-roles',
-    'accessibility/aria-valid-attr-value',
-    'accessibility/aria-valid-attr',
-    'accessibility/audio-caption',
-    'accessibility/button-name',
-    'accessibility/bypass',
-    'accessibility/color-contrast',
-    'accessibility/definition-list',
-    'accessibility/dlitem',
-    'accessibility/document-title',
-    'accessibility/duplicate-id',
-    'accessibility/frame-title',
-    'accessibility/html-has-lang',
-    'accessibility/html-lang-valid',
-    'accessibility/image-alt',
-    'accessibility/input-image-alt',
-    'accessibility/label',
-    'accessibility/layout-table',
-    'accessibility/link-name',
-    'accessibility/list',
-    'accessibility/listitem',
-    'accessibility/meta-refresh',
-    'accessibility/meta-viewport',
-    'accessibility/object-alt',
-    'accessibility/tabindex',
-    'accessibility/td-headers-attr',
-    'accessibility/th-has-data-cells',
-    'accessibility/valid-lang',
-    'accessibility/video-caption',
-    'accessibility/video-description',
+  },
+};
+
+const defaultFlags = {
+  chromeFlags: [
+    '--disable-gpu',
+    '--headless',
+    '--disable-storage-reset',
+    '--enable-logging',
+    '--disable-device-emulation',
   ],
 };
 
+/**
+ *
+ * @return {Object}
+ * @param config
+ */
+function getAccessibilityConfig(config) {
+  if (!config) {
+    config = Object.assign({}, defaultConfig);
+    config.settings.onlyCategories.push('accessibility');
```

Comment:
This still modifies `defaultConfig`, which seems bad.  Options include:

```js
config = JSON.parse(JSON.stringify(defaultConfig));
```

or

```js
const cloneDeep = require('lodash/cloneDeep');
// ...
config = cloneDeep(defaultConfig);
```

or

```js
config = Object.assign({}, config, {
  settings: Object.assign({}, config.settings, {
    onlyCategories: config.settings.onlyCategories.concat(['accessibility']),
  }),
});
```

Reply:
addressing